### DISCUSSION TITLE
使用Taro开发RN，推荐吗？

### DISCUSSION THREAD

**Discussion by wuweikd** [NervJS/taro#15875] [Q & A]
Taro是否适合开发RN，团队没相关经验的同事。
我自己总结了一些，但是可能不全，或者错了，有相关经验的前辈指点一下啊，群也加满了、、、

![image](https://github.com/NervJS/taro/assets/34428646/e255e3ad-08b9-4a30-9d32-2dc85d0741e3)

![image](https://github.com/NervJS/taro/assets/34428646/ed0431ef-490f-4db3-956b-6087ef7e4fb2)

![image](https://github.com/NervJS/taro/assets/34428646/9c4ddbb6-28ef-4359-84bb-65f719d42711)

**Comment by honlyHuang** [comment]
没有问题的哈，58 同城的同学维护 RN 端，他们也一直在使用

**Reply by dingtwo** [reply]
> 没有问题的哈，58 同城的同学维护 RN 端，他们也一直在使用

有推荐的taro用的RN组件库吗

**Reply by ShaoGongBra** [reply]
> > 没有问题的哈，58 同城的同学维护 RN 端，他们也一直在使用
> 
> 有推荐的taro用的RN组件库吗

duxui，我维护的，目前基本是唯一支持多端比较完善的ui库了  http://duxapp.cn

**Comment by btw1024** [comment]
更好的思路也许是通过编写taro插件来支持使用rn源代码开发。

打包编译阶段通过插件转换为taro的类react语法来支持小程序。

H5通过rn-web实现。
